components/x-teaser/src/concerns/image-service.js

```diff
@@ -1,5 +1,9 @@
 const BASE_URL = 'https://www.ft.com/__origami/service/image/v2/images/raw';
-const OPTIONS = ['source=next', 'fit=scale-down', 'dpr=2'];
+const OPTIONS = Object.freeze({
```

Comment:
what's the purpose of `Object.freeze` here? there's no way for this object to be mutated,, it seems unnecessary

Reply:
ok, i guess it's a bit of a future-proof